pillarbox-ui/src/main/java/ch/srgssr/pillarbox/ui/exoplayer/ExoPlayerView.kt

```diff
@@ -56,6 +60,8 @@ fun ExoPlayerView(
     controllerAutoShow: Boolean = true,
     showNextButton: Boolean = true,
     showPreviousButton: Boolean = true,
+    showShuffleButton: Boolean = false,
+    showSubtitleButton: Boolean = false,
```

Comment:
I think we should create a option class or something like that. But if we want it, it can be done in another PR.

Reply:
Good point! I've added a callback argument to `ExoPlayerView` so it's possible to access any `PlayerView` option easily.